docs/reference/release-notes/7.7.asciidoc

```diff
@@ -1,22 +1,612 @@
+* Fix caching for PreConfiguredTokenFilter {pull}50912[#50912] (issue: {issue}50734[#50734])
+
+Audit::
+* Logfile audit settings validation {pull}52537[#52537] (issues: {issue}47038[#47038], {issue}47711[#47711], {issue}52357[#52357])
+
+Authentication::
+* Fix potential bug in concurrent token refresh support {pull}53668[#53668]
+* Respect runas realm for ApiKey security operations {pull}52178[#52178] (issue: {issue}51975[#51975])
+* Preserve ApiKey credentials for async verification {pull}51244[#51244]
+* Don't fallback to anonymous for tokens/apikeys {pull}51042[#51042] (issue: {issue}50171[#50171])
+* Fail gracefully on invalid token strings {pull}51014[#51014]
+
+Authorization::
+* Explicitly require that delegate API keys have no privileges {pull}53647[#53647]
+* Allow _rollup_search with read privilege {pull}52043[#52043] (issue: {issue}50245[#50245])
+
+CCR::
+* Put CCR tasks on (data && remote cluster clients) {pull}54146[#54146] (issue: {issue}53924[#53924])
+* Handle no such remote cluster exception in ccr {pull}53415[#53415] (issue: {issue}53225[#53225])
+* Fix shard follow task cleaner under security {pull}52347[#52347] (issues: {issue}44702[#44702], {issue}51971[#51971])
+
+CRUD::
+* Force execution of finish shard bulk request {pull}51957[#51957] (issue: {issue}51904[#51904])
+* Return 429 status code when there's a read_only cluster block {pull}50166[#50166] (issue: {issue}49393[#49393])
+
+Cluster Coordination::
+* Use special XContent registry for node tool {pull}54050[#54050] (issue: {issue}53549[#53549])
+* Allow static cluster.max_voting_config_exclusions {pull}53717[#53717] (issue: {issue}53455[#53455])
+* Allow joining node to trigger term bump {pull}53338[#53338] (issue: {issue}53271[#53271])
+* Ignore timeouts with single-node discovery {pull}52159[#52159]
+
+Distributed::
+* Execute retention lease syncs under system context {pull}53838[#53838] (issues: {issue}48430[#48430], {issue}53751[#53751])
+* Exclude nested documents in LuceneChangesSnapshot {pull}51279[#51279]
+
+Engine::
+* Fix doc_stats and segment_stats of ReadOnlyEngine {pull}53345[#53345] (issues: {issue}51303[#51303], {issue}51331[#51331])
+* Do not wrap soft-deletes reader for segment stats {pull}51331[#51331] (issues: {issue}51192[#51192], {issue}51303[#51303])
+* Account soft-deletes in FrozenEngine {pull}51192[#51192] (issue: {issue}50775[#50775])
+
+Features/CAT APIs::
+* Fix NPE in RestPluginsAction {pull}52620[#52620] (issue: {issue}45321[#45321])
+
+Features/ILM+SLM::
+* Disallow negative TimeValues {pull}53913[#53913] (issue: {issue}54041[#54041])
+* Fix null config in SnapshotLifecyclePolicy.toRequest {pull}53328[#53328] (issues: {issue}44465[#44465], {issue}53171[#53171])
+* Freeze step retry when not acknowledged {pull}53287[#53287]
+* Make the set-single-node-allocation retryable (#52077) {pull}52138[#52138] (issue: {issue}52077[#52077])
+* Make the set-single-node-allocation retryable {pull}52077[#52077] (issue: {issue}43401[#43401])
```

Comment:
```suggestion
Duplicates?

Looks like #52077 is the original PR. #52138 is the 7.x backport.
* Make the set-single-node-allocation retryable (#52077) {pull}52138[#52138] (issue: {issue}52077[#52077])
* Make the set-single-node-allocation retryable {pull}52077[#52077] (issue: {issue}43401[#43401])
```

Reply:
Indeed, thanks for spotting it. Updated your suggestion and committed it.